internal/nonkube/bootstrap/uninstall.go

```diff
@@ -1,14 +1,81 @@
 package bootstrap
 
 import (
-	"github.com/skupperproject/skupper/internal/nonkube/common"
-	"github.com/skupperproject/skupper/pkg/nonkube/api"
+	"fmt"
 	"os"
+	"os/user"
 	"path"
+
+	"github.com/skupperproject/skupper/api/types"
+	"github.com/skupperproject/skupper/internal/nonkube/bootstrap/controller"
+	internalclient "github.com/skupperproject/skupper/internal/nonkube/client/compat"
+	"github.com/skupperproject/skupper/internal/nonkube/common"
+	"github.com/skupperproject/skupper/pkg/nonkube/api"
 )
 
 func Uninstall(platform string) error {
 
+	currentUser, err := user.Current()
+	if err != nil {
+		return fmt.Errorf("failed to get current user: %v", err)
+	}
+
+	containerName := fmt.Sprintf("%s-skupper-controller", currentUser.Username)
+
+	isContainerAlreadyRunningInPodman, err := IsContainerRunning(containerName, types.PlatformPodman)
```

Comment:
Similarly, we should ignore errors in case platform is not available (as docker and podman might now always be installed on the system).

Reply:
done